File: Java/Kode/Guitar.java
public class Guitar extends MusicInstrument {
    private String brand; // Yamaha / Fender / Jackson / Schecter
    private String series; // mis. "Pacifica112"
    private int fretSize; // 24 / 22 / 15 / 12 / 19
    private String stringType; // nilon / nikel / baja / bronze

    // constructor kosong
    public Guitar() {
        super();
        this.brand = "";
        this.series = "";
        this.fretSize = 0;
        this.stringType = "";
    }

    // constructor berparameter (memanggil constructor MusicInstrument lewat super())
    public Guitar(int id, double price, int stock,
                  String playingMethod, String condition, String weight,
                  String brand, String series, int fretSize, String stringType) {
        super(id, price, stock, playingMethod, condition, weight);
        this.brand = brand;
        this.series = series;
        this.fretSize = fretSize;
        this.stringType = stringType;
    }

    // getter and setter brand
    public void setBrand(String brand) { this.brand = brand; }
    public String getBrand() { return brand; }

    // getter and setter series
    public void setSeries(String series) { this.series = series; }
    public String getSeries() { return series; }

    // getter and setter fretSize
    public void setFretSize(int fretSize) { this.fretSize = fretSize; }
    public int getFretSize() { return fretSize; }

    // getter and setter stringType
    public void setStringType(String stringType) { this.stringType = stringType; }
    public String getStringType() { return stringType; }
}

File: Python/Kode/Guitar.py
from MusicInstrument import MusicInstrument

class Guitar(MusicInstrument):
    def __init__(self, id=0, price=0.0, stock=0,
                 playing_method="", condition="", weight="",
                 brand="", series="", fret_size=0, string_type=""):
        super().__init__(id, price, stock, playing_method, condition, weight)
        self.brand = brand
        self.series = series
        self.fret_size = fret_size
        self.string_type = string_type

    # getter and setter brand
    def set_brand(self, brand):
        self.brand = brand

    def get_brand(self):
        return self.brand

    # getter and setter series
    def set_series(self, series):
        self.series = series

    def get_series(self):
        return self.series

    # getter and setter fretSize
    def set_fret_size(self, fret_size):
        self.fret_size = fret_size

    def get_fret_size(self):
        return self.fret_size

    # getter and setter stringType
    def set_string_type(self, string_type):
        self.string_type = string_type

    def get_string_type(self):
        return self.string_type